_commands/os_x/pbcopy.md

```diff
@@ -0,0 +1,55 @@
+---
+---
+
+pbcopy
+-------
+
+`pbcopy` is used to copy text from the terminal into the clipboard.
```

Comment:
Can you cross-link (or since I don't think it's merged in yet – #132) `xclip` here?